README.md

```diff
@@ -99,32 +101,28 @@ Once your pull request has been merged, the site is deployed automatically.
 1. Ensure you first edit `_data/authors.yml` to add you information
 2. Create a new ASCII Doc file under `_posts/blog`. The file should be named with the following format: `yyyy-MM-DD-title-in-lowercase.adoc`
 3. Add a header in your file. Modify the fields according to your post, especially for the title, headline, tags, authors and date
-```
----
-layout: post
-title: "Your blog post title"
-headline: "Your blo post headline"
-categories: blog
-hidden: false
-author: your_author_key
-tags: [ehcache, website]
-date: 2016-05-04T18:00:00-05:00
-modified:
----
-```
+   ```
+   layout: post
```

Comment:
The `---` are normal. They are the front matter delimiters.

Reply:
K ... I'll put those back.  The indentation will remain since that's the proper way using MarkDown to continue content.

Reply:
Sure.